src/Paket.Bootstrapper/ArgumentParser.cs

```diff
@@ -276,8 +276,9 @@ private static string GetMagicModeTarget(IFileSystemProxy fileSystemProxy)
             //    accessing the same file as their path depends on the bootstrapper path.
             var assemblyLocation = fileSystemProxy.GetExecutingAssemblyPath();
             var targetName = String.Format("paket_{0}.exe",GetHash(assemblyLocation));
-
-            return Path.Combine(fileSystemProxy.GetTempPath(), targetName);
+            var targetFolder = Path.Combine(fileSystemProxy.GetTempPath(), "paket");            
+            Directory.CreateDirectory(targetFolder);
```

Comment:
Please use `fileSystemProxy` here (it should have a `CreateDirectory` method too)

Reply:
Done 👍🏼